bug(frontend): CacheService.ts TS2352 — unsafe cast of Window to Record<string, unknown>

## Discovery Source
Found during implementation of #4537 — agent noted pre-existing error in CacheService.ts.

## Problem
`autobot-frontend/src/services/CacheService.ts:202` has:

```
error TS2352: Conversion of type 'Window & typeof globalThis' to type 'Record<string, unknown>' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
```

A direct cast from `Window & typeof globalThis` (the `window` object) to `Record<string, unknown>` is rejected by TypeScript because the types don't sufficiently overlap. This is an unsafe cast that could hide runtime errors.

## Fix
Use a double cast through `unknown` to make intent explicit:
```typescript
(window as unknown as Record<string, unknown>)
```
Or refactor to avoid the cast entirely by accessing the specific window property with proper typing.

## Files
- `autobot-frontend/src/services/CacheService.ts` line 202

--- a/autobot-frontend/src/services/CacheService.ts
+++ b/autobot-frontend/src/services/CacheService.ts
@@ -199,7 +199,7 @@ export const cacheService = new CacheService();
 
 // Make available globally for debugging
 if (typeof window !== 'undefined') {
-  (window as Record<string, unknown>).cacheService = cacheService;
+  (window as unknown as Record<string, unknown>).cacheService = cacheService;
 }
 
 export default cacheService;

